docs/pages/world/modules.mdx

```diff
@@ -1,17 +1,241 @@
+import { CollapseCode } from "../../components/CollapseCode";
+
 # Modules
 
 Modules are onchain installation scripts that create resources and their associated configuration when called by a `World`.
 This is somewhat similar to one of the use cases for [foundry scripts](https://book.getfoundry.sh/tutorials/solidity-scripting), except that modules are deployed onchain and can be used by any `World` on the same chain.
 
 ## Module installation
 
+The easiest way to install modules is to edit [the config file](/config/reference#modules).
+
+For example, here is a modified config file that installs [the ERC-721 module](/world/modules/erc721).
+
+<CollapseCode>
+
+```typescript filename="mud.config.ts" copy {2,14-50}
+import { defineWorld } from "@latticexyz/world";
+import { encodeAbiParameters } from "viem";
+
+export default defineWorld({
+  namespace: "app",
+  tables: {
+    Counter: {
+      schema: {
+        value: "uint32",
+      },
+      key: [],
+    },
+  },
+  modules: [
+    {
+      artifactPath: "@latticexyz/world-modules/out/PuppetModule.sol/PuppetModule.json",
+      root: false,
+      args: [],
+    },
+    {
+      artifactPath: "@latticexyz/world-modules/out/ERC721Module.sol/ERC721Module.json",
+      root: false,
+      args: [
+        {
+          type: "bytes",
+          value: encodeAbiParameters(
+            [
+              { type: "bytes14" },
+              {
+                type: "tuple",
+                components: [{ type: "string" }, { type: "string" }, { type: "string" }],
+              },
+            ],
+            ["0x44444444".padEnd(30, "0"), ["No Valuable Token", "NVT", "http://www.example.com/base/uri/goes/here"]],
+          ),
```

Comment:
I think this would be easier to read if we lifted it up into a variable rather than embedding it here.

```ts
const erc721ModuleArgs = encodeAbiParameters(...);

// then in the config
args: [{ type: "bytes", value: erc721ModuleArgs }]
```

Reply:
Good idea, done.